src/applications/vre/28-8832/wizard/pages/VRECounselorNotification.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const VRECounselorNotification = () => (
+  <div className="vads-u-margin-top--2 vads-u-background-color--primary-alt-lightest vads-u-padding--3">
+    <p className="vads-u-margin-top--0">
+      Please contact your vocational rehabilitation counselor to learn more
+      about how to get Chapter 31 career planning and guidance benefits.
+    </p>
+    <a
+      href="https://www.benefits.va.gov/benefits/offices.asp"
+      target="_blank"
+      rel="noopener noreferrer"
```

Comment:
Do we need this `rel` for links in the same domain?

Reply:
linter throws a warning when using `target=_'blank'` without `rel`..but since you mention the domain thing, it might just be better to get rid of the _blank altogether.